python/tests/test_lowlevel.py

```diff
@@ -408,6 +408,37 @@ def test_equals_bad_args(self):
         with pytest.raises(TypeError):
             tc.equals(tc, ignore_timestamps=bad_bool)
 
+    def test_asdict(self):
+        for ts in self.get_example_tree_sequences():
+            tc = _tskit.TableCollection(sequence_length=ts.get_sequence_length())
+            ts.dump_tables(tc)
+            d = tc.asdict()
+            # Method is tested extensively elsewhere, just basic sanity check here
+            assert isinstance(d, dict)
+            assert len(d) > 0
+
+    def test_fromdict(self):
+        for ts in self.get_example_tree_sequences():
+            tc1 = _tskit.TableCollection(sequence_length=ts.get_sequence_length())
+            ts.dump_tables(tc1)
+            d = tc1.asdict()
+            tc2 = _tskit.TableCollection(sequence_length=0)
+            tc2.fromdict(d)
+            assert tc1.equals(tc2)
+
+    def test_asdict_bad_args(self):
+        ts = msprime.simulate(10, random_seed=1242)
```

Comment:
You could use `simple_degree1_ts_fixture` here, but I don't feel strongly about it.

Reply:
I was following the conventions in the rest of the class, so I don't think it's worth changing to a fixture here.